App/Services/DeepLink.ts

```diff
@@ -2,6 +2,8 @@ import NavigationService from './NavigationService'
 import Config from 'react-native-config'
 import { ExternalInvite, DeepLinkData } from '../Models/TextileTypes'
 
+const DEBUG_PATH_PREFIX = '/debug'
```

Comment:
Would it be more simple to just support another url prefix, like `debug.textile.io`? Then we wouldn't have to support any special cases in the code like this.

Reply:
You'd still need to detect it in the code, yeah? How could that work different?

Reply:
Haven't thought very deeply about this, but was imagining turning this into more of a configuration/environment thing. If the app is configured to create/process URLs of a specific prefix, that prefix being part of the .env or whatever, then everything should just work seamlessly and consistently, no special cases. It's related to a larger idea of having a debug/staging environment at all levels of the system we're building.

Reply:
we could do that really nicely by leveraging the .env (say .env.debug to standardize) files instead of this and still using that config in the deeplink.ts file. that way it would work seamlessly for both ios/android. i didn't do that this time around because i wasn't sure we had a rigid practice of .env yet.